packages/app-shortcut/ios/Plugin/AppShortcutPlugin.swift

```diff
@@ -7,12 +7,64 @@ import Capacitor
  */
 @objc(AppShortcutPlugin)
 public class AppShortcutPlugin: CAPPlugin {
+    public let tag = "AppShortcuts"
+    public static let onAppShortcutEvent = "onAppShortcut"
+    public static let userInfoShortcutItemKey = "shortcutItem"
     private let implementation = AppShortcut()
 
-    @objc func echo(_ call: CAPPluginCall) {
-        let value = call.getString("value") ?? ""
-        call.resolve([
-            "value": implementation.echo(value)
+    override public init() {
+        super.init()
+        NotificationCenter.default.addObserver(self, selector: #selector(self.handleAppShortcutNotification), name: NSNotification.Name(AppShortcutPlugin.onAppShortcutEvent), object: nil)
+    }
+
+    deinit {
+        NotificationCenter.default.removeObserver(self)
+    }
+
+    @objc func get(_ call: CAPPluginCall) {
+        implementation.get(completion: { result in
+            call.resolve(result.toJSObject() as! JSObject)
+        })
+    }
+
+    @objc func set(_ call: CAPPluginCall) {
+        do {
+            let options = try SetOptions(call: call)
+            implementation.set(shortcuts: options.getShortcuts, completion: { error in
+                if let error = error {
+                    CAPLog.print("[", self.tag, "] ", error)
+                    call.reject(error.localizedDescription)
+                    return
+                }
+                call.resolve()
```

Comment:
```suggestion
                if let error = error {
                    self.rejectCall(call, error)
                } else {
                    self.resolveCall(call)
                }
```

Please copy the `rejectCall` and `resolveCall` methods from [here](https://github.com/capawesome-team/capacitor-plugins/blob/c3d20a436dc2fa9dfdcecd802af7677ddfeda1a8/packages/app-review/ios/Plugin/AppReviewPlugin.swift#L44) (sorry, I forgot to mention that the first time). I really like that approach because it looks way cleaner. Please also apply it to all other methods. The same also exists for Android.